src/frontends/onnx/frontend/include/onnx_import/core/null_node.hpp

```diff
@@ -44,7 +26,7 @@ namespace onnx_import {
 ///
 /// More:
 /// https://github.com/onnx/onnx/blob/master/docs/IR.md#optional-inputs-and-outputs
-class NGRAPH_API_DEPRECATED ONNX_IMPORTER_API NullNode : public ov::op::Op {
```

Comment:
This API was deprecated.  And now we allow to use it again. Is it ok?
Maybe previously we planned to move it to dev API or smth similar?